src/wp-includes/css/custom-properties.css

```diff
@@ -101,6 +101,243 @@ body {
+
+	--wp-admin--customize-add-new-widget--background: #f0f0f1;
+	--wp-admin--customize-add-new-widget--border-color: #8c8f94;
+	--wp-admin--customize-add-new-widget--box-shadow-color_inset: rgb(0, 0, 0);
+	--wp-admin--customize-add-new-widget--color: #2c3338;
+
+	--wp-admin--customize-accordion-section-dropdown-content--border-color: #f0f0f1;
+	--wp-admin--customize-accordion-section-installed_themes--border-top-color: #dcdcde;
+	--wp-admin--customize-accordion-section-themes-control-section--border-top-color: #dcdcde;
+
+	--wp-admin--customize-available-menu-items-search-icon--color: #646970;
+	--wp-admin--customize-available-widgets--background: #f0f0f1;
+	--wp-admin--customize-available-widgets--border-right-color: #dcdcde;
+	--wp-admin--customize-available-widgets--border-top-color: #dcdcde;
+	--wp-admin--customize-available-widgets-filter--background: #f0f0f1;
+	--wp-admin--customize-available-widgets-filter-clear-results--background: none;
+	--wp-admin--customize-available-widgets-filter-clear-results--box-shadow-color_first-outer-box--focus: #4f94d4;
+	--wp-admin--customize-available-widgets-filter-clear-results--box-shadow-color_second-outer-box--focus: rgba(79, 148, 212, 0.8);
+	--wp-admin--customize-available-widgets-filter-clear-results--color: #d63638;
+	--wp-admin--customize-available-widgets-filter-clear-results--color--hover: #d63638;
+	--wp-admin--customize-available-widgets-template--background: #fff;
+	--wp-admin--customize-available-widgets-template--border-bottom-color: #dcdcde;
+	--wp-admin--customize-available-widgets-template--border-left-color: #fff;
+	--wp-admin--customize-available-widgets-top--background: transparent;
+	--wp-admin--customize-available-widgets-section-title-small-screen--color: #50575e;
+
+	--wp-admin--customize-button-add-media--background-color: #f0f0f1;
+	--wp-admin--customize-button-add-media--background-color--hover: #fff;
+	--wp-admin--customize-button-add-media--background-color--focus: #fff;
+	--wp-admin--customize-button-add-media--border-color--focus: #3582c4;
+	--wp-admin--customize-button-add-media--box-shadow-color--focus: #3582c4;
+	--wp-admin--customize-button-add-media--color: #2c3338;
+
+	--wp-admin--customize-control-changeset-inside-control-row--background-color: #fff;
+	--wp-admin--customize-control-changeset-inside-control-row--border-bottom-color: #dcdcde;
+	--wp-admin--customize-control-changeset-inside-control-row--border-top-color--first-of-type: #dcdcde;
+	--wp-admin--customize-control-changeset_preview_link-input--border-top-color: #dcdcde;
+	--wp-admin--customize-control-changeset_preview_link-input--color: #fff;
+	--wp-admin--customize-control-changeset_preview_link-disabled--color: #000;
+	--wp-admin--customize-control-changeset_scheduled_date--background: #fff;
+	--wp-admin--customize-control-changeset_scheduled_date--border-bottom-color: #dcdcde;
+
+	--wp-admin-customize-control-color-dropdown-content--border-color--hover: rgb(0, 0, 0);
```

Comment:
This should be a hex value - we only use `rgb` when the color has opacity.

Reply:
Excellent feedback! 

I'm currently away from home to manage the affairs of my elderly father. I expect to return home ( St. Louis, MO ) by Friday. I need a block of time to sit down with your comments and my code editor, and modify my PR to conform to your comments. 

I'll have an update for you next week.

Reply:
hey @rgadon107, just checking in. we're human first and totally understand how challenging it can be to juggle family responsibilities and life challenges. thanks for all your hard work on this, wasn't a small undertaking!

Could you circle back with me sometime in the next 5 days just with a status update and ETA? 

In the interest of getting this project merge before it gets too stale we're trying to shepherd along. If you'd like one of us to pick up the baton that's totally fine and also totally understand if you'd like to carry it across the finish line. If we don't hear back we'll probably work off this branch starting later next week.

Reply:
Hi @0aveRyan! My apologies for not following through and updating the PR per the recent feedback from @ryelle. **I graciously yield this PR to another member of the team to complete.** I'm managing a lot of non-WordPress/non-webdev responsibilities right now, and can't give this PR the time and energy that it deserves. Thank you for reaching out and asking about my status. I hope that my contribution on this PR makes it easier for others to edit into it's final form.